firebase-firestore/src/main/java/com/google/firebase/firestore/AggregateQuerySnapshot.java

```diff
@@ -0,0 +1,50 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.firebase.firestore;
+
+import androidx.annotation.Nullable;
+import java.util.Objects;
+
+public class AggregateQuerySnapshot {
```

Comment:
Please add class-level javadocs to AggregateQuerySnapshot and javadocs for getCount().